README.md

```diff
@@ -145,6 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Verscend Technologies](http://verscend.com.np/) | Hattisar, Kathmandu | Software research and development center
 - [Vianet Communications Pvt. Ltd.](https://www.vianet.com.np/) | Jawalakhel, Lalitpur | Provider of high speed Internet connectivity and communications solutions for business and individuals
 - [Visualise VR](http://visualisevr.com/) | Pulchowk, Lalitpur |  Help business benefit from mixed reality technologies
+- [Viveka Health](http://www.vivekahealth.com/) | Sanepa, Lalitpur | Outsourcing company
```

Comment:
Hello @SushilShrestha, `Outsourcing company` looks a bit generic. Can we add a proper description?

Reply:
Sure I will update those.